Modules/Core/QuadEdgeMesh/test/itkQuadEdgeMeshEulerOperatorDeleteCenterVertexTest.cxx

```diff
@@ -82,7 +82,13 @@ itkQuadEdgeMeshEulerOperatorDeleteCenterVertexTest(int argc, char * argv[])
   }
   std::cout << "OK" << std::endl;
 
-  (void)deleteCenterVertex->GetNameOfClass();
+  const std::string className = deleteCenterVertex->GetNameOfClass();
+  const std::string requiredClassName{ "QuadEdgeMeshEulerOperatorDeleteCenterVertexFunction" };
+  if (className != requiredClassName)
+  {
+    std::cout << className << " != " << requiredClassName << std::endl;
+    return EXIT_FAILURE;
+  }
```

Comment:
@hjmjohnson @jhlegarreta @thewtex I'm sorry but that's really a lot of code for such a trivial test. Especially now that we have more than a thousand GetNameOfClass() overloads and overrides. Please consider using `ITK_TEST_EXPECT_EQUAL` instead. Or GoogleTest `EXPECT_EQ`

Reply:
@N-Dekker agreed. A new PR can certainly be made to address that.

Reply:
> A new PR can certainly be made to address that.

@jhlegarreta Please feel free to make that a new PR  😃

Reply:
@N-Dekker PR #4583.

Reply:
Thanks @jhlegarreta Sorry I'm too late to review your pull request #4581. I did leave the computer off this weekend  🤷  But I'm glad to see your pull request effectively addressed the duplicate test code 👍